### DISCUSSION TITLE
Why is the individual container docker setup preferred over the AIO container? Where are the docker-compose installation instructions?

### DISCUSSION THREAD

**Discussion by edgariscoding** [hoppscotch/hoppscotch#3560] [Q&A] [answered]
While reading over the setup instructions I noticed that there's instructions for individual containers and it's marked as recommended.

Then there's an AIO container. What's the reasoning for this? Why is one recommended over the other? What are the drawbacks of AIO?

I also noticed that there's a docker-compose file in the repo but there's no mention about it in the instructions, unless I missed it. 
Is the docker-compose file working?

**Comment by Tolk2402** [comment, marked as answer]
Absolutely a working guide here: https://medium.com/@kiranjohns/self-host-your-api-testing-with-hoppscotch-the-open-source-api-platform-cbe8b62d3c54

The funny thing is that the guide from the official documentation turned out to be completely useless for me, the "docker run" command did not work, giving an error like “local image not found or authorization required,” so I was able to configure it locally only through "docker compose"
